### ISSUE TITLE
[1.9.4-PE1.0.4B] Setting Custom EMC ignore Types

### ISSUE THREAD

**Issue by SoraHjort** [sinkillerj/ProjectE#1301]
TL;DR: Custom EMC doesn't work for items that use Types instead of Damage to differentiate multiple items with the same ID.

------

It seems setting a custom EMC on a item uses just the meta. But I've ran into an issue with some mods that use Type tags. An example for reference, would be Botania. Botania has a series of items called botania:specialFlower. These are for things such as Dayblooms, Nightshades, and Pure Daisies. They have a meta of 0, and are defined by the type tag. 

Because of this, setting a value to one will set the value for all of them. It also results in you only being able to learn one of them. It'll let you learn any of them for transmutation, but you won't beable to learn any more than just the one. So you could learn the Nightshade, but not the Daybloom or Pure Daisy. Or you could learn the Pure Daisy, and none of the others.  It becomes a first learn first only situation. It's a situation of having to manually edit the player file to add in the items. 

I don't know if this'll be something that would be fixed, since I'm unfamiliar beyond knowing enough to fix player files when it comes to the NBT structure. But I've included an image as an example to show what I meant by the Type tag for two of the items. 
![botaniaexample](https://cloud.githubusercontent.com/assets/18604504/16906738/f7211882-4c68-11e6-9247-26ab3d5005c0.png)

**Comment by escman1999** [comment]
projecte has trouble with items that use NBT data

**Comment by MaPePeR** [comment]
There is no real way of doing NBT data right. All mods use it for total different kind of purposes. You have a (Non-ProjectE-)bag in your inventory, that stores items? Probably uses NBT for that. 
Picked up a Machine and it kept its inventory/RF/configuration? That is NBT. Charged your Klein Star with EMC? That is NBT, too.

So you can never really know if the NBT stores a state of an item/block or if it is actually important for the type of the item.

**Comment by sinkillerj** [comment]
Known issue
